src/server/qgshttprequesthandler.cpp

```diff
@@ -505,6 +511,8 @@ bool QgsHttpRequestHandler::startGetFeatureResponse( QByteArray* ba, const QStri
     format = "text/xml";
 
   setInfoFormat( format );
+  startStreamingResponse();
```

Comment:
All "GetFeature"-requests. As I understand the code the response for GetFeature-requests are always streamed